sa/sa.go

```diff
@@ -333,14 +341,47 @@ func (t TooManyCertificatesError) Error() string {
 // certificates than that matching one of the provided domain names, it will return
 // TooManyCertificatesError.
 func (ssa *SQLStorageAuthority) CountCertificatesByNames(ctx context.Context, domains []string, earliest, latest time.Time) ([]*sapb.CountByNames_MapElement, error) {
```

Comment:
Can you update the comment for this function to mention the parallelism? It also might be worth mentioning that the first error result returns immediately.